Packs/AMP/Integrations/integration-AMP.yml

```diff
@@ -114,6 +121,25 @@ script:
     var getURL = function(command, args) {
         var base = urlDict[command];
         switch (command) {
+
+            case 'amp_delete_computers_isolation':
+                base += '/' + args['connector_guid'] + '/isolation';
+                delete args['connector_guid'];
+                return base
```

Comment:
why is the return here needed? we return `base` at the end of the switch case statement.
```suggestion
```